apps/pom.xml

```diff
@@ -727,7 +734,7 @@
                     <configuration>
                         <toolchains>
                             <jdk>
-                                <!-- Use a Java 17 JDK, by default. -->
+                                <!-- Use a Java 21 JDK, by default. -->
```

Comment:
Maybe just nix this comment so we don't have to keep updating it. The version is defined in the variable.

Reply:
ok. We will do